python/README.md

```diff
@@ -0,0 +1,129 @@
+# Code Snippets : Python
+Code snippets in Python demonstrating how to use various modules of the [MediaWiki Action API](https://www.mediawiki.org/wiki/API:Main_page)
```

Comment:
Also add the demo apps in here.

Reply:
I was unsure whether I needed to add Demo Apps in the README of python or just include "demo apps" in the text. So, I added both.